adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java

```diff
@@ -146,6 +153,7 @@ public AdditionalInfo getInfo(final Principal clientIdentity, final Object custo
                 final String deviceId = (String) customArgument;
                 final PreSharedKeyDeviceIdentity deviceIdentity = getHandshakeIdentity(clientIdentity.getName(), null);
                 result.put("hono-device", new Device(deviceIdentity.getTenantId(), deviceId));
+                result.put("hono-auth-id", deviceIdentity.getAuthId());
```

Comment:
I guess it would be good to define constants for these keys as they are used across different classes